src/Microsoft.DotNet.Wpf/src/Shared/MS/Win32/HwndWrapper.cs

```diff
@@ -95,10 +95,11 @@ public HwndWrapper(
             // Register will fail if the string gets over 255 in length.
             // So limit each part to a reasonable amount.
             string appName;
-            if(null != AppDomain.CurrentDomain.FriendlyName && 128 <= AppDomain.CurrentDomain.FriendlyName.Length)
-                appName = AppDomain.CurrentDomain.FriendlyName.Substring(0, 128);
+            var currentDomainFriendlyName = AppDomain.CurrentDomain.FriendlyName;
```

Comment:
Could we not use `var` here?

Reply:
@lindexi can you take a look at this ?

Reply:
OK, I update it. Thank you @h3xds1nz and @dipeshmsft